src/SlowSlothBudget.Web/Controllers/ExpensesController.cs

```diff
@@ -58,6 +61,12 @@ public IActionResult GetExpenses(string category, string description, int offset
                 Offset = offset,
                 Limit = limit
             }, out var totalUserExpensesNumber);
+
+            //BART: any reason this is passed in header instead of just an aggregate object? like
+            /*
+             * Total: int
+             * Expenses: List<Expenses>
+             */
```

Comment:
Yes, I followed an advice from [some REST API best practices article](https://blog.mwaysolutions.com/2014/06/05/10-best-practices-for-better-restful-api/), point 7. -> paging.